Makefile

```diff
@@ -0,0 +1,3 @@
+test:
+	@echo "  >  Running unit tests"
+	go test ./...
```

Comment:
missing new line

perhaps add lint also, as it was added to yml

Reply:
added